draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/NavigationTabs.tsx

```diff
@@ -3,6 +3,13 @@ import classnames from "classnames"
 import { NON_REVERSED_VARIANTS, Variant } from "./TitleBlockZen"
 import styles from "./NavigationTabs.module.scss"
 
+export type CustomNavigationTabProps = Omit<
+  NavigationTabProps,
+  "component" | "variant"
+> & {
+  className: string
+}
```

Comment:
What if we ditched the href!!

Reply:
Don't you want it for the other side so that you can do `<Link to={props.href} />`?

Reply:
Once we're in userland, the user can do what they want. I guess if they're passing the component by reference like in our test they might want it but personally I'd do this:
```
component: (props) => <Link to={IControlMyOwnDestiny}>{AndMyChildrensDestiny}</Link>
```

`className` is almost the _only_ thing we want passed. It's the only thing a custom component shares with a default component.

Reply:
Maybe we should use a discriminated union with the component prop and everything else? If you are going to use the renderProp API, you _cannot_ pass the other props.